src/estimation/src/mekf6.cpp

```diff
@@ -65,7 +65,7 @@ Mekf6::GetQMatrix(double dt) {
   Q.block<3, 3>(6, 12) = -m_sigma_beta_f * dt * dt * dt / 6.0f;
 
   Q.block<3, 3>(9, 0) = -m_sigma_beta_omega * dt * dt / 2.0f;
-  Q.block<3, 3>(9, 9) = m_sigma_beta_omega * dt * dt / 2.0f;
+  Q.block<3, 3>(9, 9) = m_sigma_beta_omega * dt;
```

Comment:
we need to verify if it's the error in the paper or in the python reference code, but the difference should be minor

Reply:
This one is following the python code. The paper by James M Maley doesnt provide Q for 6 axis, only 9 axis, so have to derive. Is your version from another paper?